### PR TITLE
Server Side Events

### PR DESCRIPTION
**Pull request by akilism**
updates the client with server side events instead of polling. discreet channels for each location in the config.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -3,12 +3,22 @@ import Form from './form';
...
-import {showLogs, fadeMarkers} from './logs';
+import { get } from './util';
+import { showLogs, fadeMarkers } from './logs';
 
 const errs = [];
 const updateInterval = 5000; // ms
 
+/* server sent events coooooode */
+console.log('setup event source');
+const logSource = new EventSource(SSE_URL);
```
Comment (broskoski) [inline src/main.js:14]: God bless you for doing this. 

Question: where is `SSE_URL` defined in here?

Comment (broskoski) [reply src/main.js:14]: Nevermind, I see below in map.html.

File: templates/map.html [hunk 2]
```diff
@@ -61,6 +61,7 @@ <h2>How to help</h2>
 const MAP_CENTER = {{ conf.MAP['CENTER'] }};
 const MAP_ZOOM = {{ conf.MAP['ZOOM'] }};
 const VERSION = '{{ config.VERSION }}';
+const SSE_URL = '{{url_for("sse.stream", channel=location)}}'
```
Comment (broskoski) [inline templates/map.html:64]: And this is going to be an actual IP address (not `localhost`)?

Comment (broskoski) [reply templates/map.html:64]: (Sorry, just realizing I might have come across short here. I was talking to @frnsys at the same time and pondering how they would set up the Redis server).

So cool, will take so much load off the server +++

Comment (akilism) [reply templates/map.html:64]: yeah it should resolve to a relative url like `/something/`;

**Pull request merged**